homeassistant/components/mobile_app/webhook.py

```diff
@@ -681,8 +688,10 @@ async def webhook_get_config(hass, config_entry, data):
 @validate_schema({vol.Required("tag_id"): cv.string})
 async def webhook_scan_tag(hass, config_entry, data):
     """Handle a fire event webhook."""
+    # Importing tag via hass.components in case it is overridden
+    # in a custom_components (custom_components.tag)
+    tag: TagProtocol = hass.components.tag
```

Comment:
I do not agree with this change. This makes no sense to me at this point.

Why are we loading a core component this weird, but we don't do the same for things like camera?

Reply:
Original discussion for mqtt and esphome.
Should those now be adjusted to use local imports instead?

Reply:
#64539

Reply:
Resolved in #82161